Add timeout and retries settings to integratord configuration (#6442)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/integration.rst

| **Allowed values** | PagerDuty/VirusTotal/Maltiverse Api key |
+--------------------+-----------------------------------------+

alert_format
^^^^^^^^^^^^

> This writes the alert file in the JSON format. The Integrator makes use of this file to fetch fields values.

Reviewer: ```suggestion
This writes the alert file in the JSON format, which the Integrator uses to fetch fields values.
```

+--------------------+-----------------------------------------------------------+
| **Default value**  | n/a                                                       |
+--------------------+-----------------------------------------------------------+
| **Allowed values** | json                                                      |
+--------------------+-----------------------------------------------------------+

> .. note:: This option must be set to ``json`` for Slack, VirusTotal, Shuffle, and Maltiverse integrations.

Reviewer: ```suggestion
.. note:: Set this option as ``json`` for Slack, VirusTotal, Shuffle, and Maltiverse integrations.
```

| **Allowed values** | json                                                      |
+--------------------+-----------------------------------------------------------+

timeout
^^^^^^^

The timeout (in seconds) to wait for a valid response from the external integration server.

+--------------------+-----------------------------------------------------------+
| **Default value**  | 10                                                        |
+--------------------+-----------------------------------------------------------+
| **Allowed values** | Any positive integer.                                     |
+--------------------+-----------------------------------------------------------+

retries
^^^^^^^

> The number of attempts to retry a request to the external integration server in case it times out.

Reviewer: ```suggestion
The maximum number of retry attempts for a request to the external integration server on timeout.
```
